README.rst

```diff
@@ -137,6 +181,27 @@ Here is the quickest way to build via a cmd/bash shell
   # To edit TeleSculptor code, open the
   # build/external/telesculptor-build/TeleSculptor.sln
 
+This will produce an installer file which can install TeleSculptor in the same
+way as the downloadable installer, as detailed in the next section.
+
+Installation
+------------
+Once you have an installer downloaded or built, you need to run it to install
```

Comment:
These directions are specific to the Linux self extracting installer.  To cover all bases we should talk about the others

On Windows run the installer executable (exe) and follow the prompts in the installer dialog.  Administrative permission is required.  On Mac, open the disk image (dmg), accept the license terms, then drag the TeleSculptor application into the Applications folder.  On Linux, run the self extracting installer script (sh) ...

Reply:
I'll include this. Do we additionally need to update the section `Before building on Linux systems...` to include instructions for installing dependencies on other OS or are they not required?

Reply:
We should try to enumerate dependencies on other platforms at some point and describe how to install them, but it's not critical right now.  Generally the dependencies are fewer on other platforms.  You need your build tools (e.g. Visual Studio) and you probably need to install Python with docutils if you want the documentation.  However, there is no single standard way to install these things on Windows and Mac.

Reply:
I think multi-platform installation is addressed better now.